specification/e-referrals-service-api.yaml

```diff
@@ -496,7 +496,44 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /R4/v1/PractitionerRole:
+    get:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to retrieve a list of available e-RS roles (i.e. business functions) at each organisation where the user works.
+
```

Comment:
Also, are Roles and Business Functions really synonymous?

Reply:
Same as above

Reply:
Ok. but I predict this is going to cause confusion.